### PR TITLE
Created second view for quiz page

### PR DESCRIPTION
**Pull request by dmtrth25**
Implemented the second view questions for quiz page

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/8abbdb9e-6209-47c3-8827-a9bbde964aed

Resolved issue: #1155

### CODE REVIEW

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const questionList = questions.map((question, index) => (
+    <QuizQuestion index={index} key={question._id} question={question} />
+  ))
+
+  return (
+    <Box>
+      <Typography sx={styles.description}>
+        Lorem ipsum dolor sit amet consectetur. Elementum at diam vulputate
+        suspendisse nulla tortor tempor odio tortor. Eu eleifend quam eget et
+        nulla.
```
Comment (MariaAfonina) [inline src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:18]: Maybe you should move this test to translations, what do you think?

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { Box, Typography } from '@mui/material'
+import QuizQuestion from '~/containers/my-quizzes//scroll-question/ScrollQuestions'
+
+import { QuizViewProps } from '~/types'
+import { styles } from '~/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles'
+
+const ScrollQuestionsQuizView: FC<QuizViewProps> = ({ questions }) => {
+  const questionList = questions.map((question, index) => (
+    <QuizQuestion index={index} key={question._id} question={question} />
+  ))
```
Comment (MariaAfonina) [inline src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:11]: If there are no questions? What will be with view tabs? Maybe you should add some checks here or on 20 lines when you render it.

File: src/containers/my-quizzes/view-quiz-container/ViewQuizContainer.tsx [hunk 3]
```diff
@@ -12,7 +13,8 @@ const ViewQuizContainer = ({ questions }: QuizContentProps) => {
       <Typography sx={styles.title}>
         Lorem ipsum dolor sit amet consectetur
```
Comment (MariaAfonina) [inline src/containers/my-quizzes/view-quiz-container/ViewQuizContainer.tsx:14]: Maybe you should move this test to translations?

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx (hunk 1 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:18]: > Maybe you should move this test to translations, what do you think?

Is it possible translate Lorem ipsum text ? It's temporary here according to design

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx (hunk 2 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:11]: > If there are no questions? What will be with view tabs? Maybe you should add some checks here or on 20 lines when you render it.

FYI, now View tab disabled if we don't have any questions, your suggestion is not correct..

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx (hunk 1 above)
Comment (MariaAfonina) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:18]: Yes, but in the future we will change this to the correct text. You can move this text to 'en' for now

File: src/containers/my-quizzes/view-quiz-container/ViewQuizContainer.tsx (hunk 3 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/view-quiz-container/ViewQuizContainer.tsx:14]: > Maybe you should move this test to translations?

the same as above. Is it possible to translate it ?

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx (hunk 1 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:18]: > Yes, but in the future we will change this to the correct text. You can move this text to 'en' for now

When we change the text and only then we can add it to 'en'. 
I suppose it is unnecessary to add random text to 'en'

Comment (MariaAfonina) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx:18]: Maybe you are right.

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const styles = {
+  description: {
+    color: 'primary.600',
+    mt: '8px',
+    mb: '32px'
```
Comment (Tolik170) [inline src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles.ts:5]: m: '8px 0 32px 0'

File: src/containers/my-quizzes/scroll-question/ScrollQuestions.styles.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  root: {
+    marginBottom: '32px'
```
Comment (Tolik170) [inline src/containers/my-quizzes/scroll-question/ScrollQuestions.styles.ts:5]: mb

File: src/containers/my-quizzes/scroll-question/ScrollQuestions.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  FormGroup,
+  RadioGroup,
+  FormControlLabel
+} from '@mui/material'
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+
+import { QuizQuestionProps } from '~/types'
+import { styles } from '~/containers/my-quizzes/scroll-question/ScrollQuestions.styles'
+
+const QuizQuestion: FC<QuizQuestionProps> = ({ question, index }) => {
```
Comment (Tolik170) [inline src/containers/my-quizzes/scroll-question/ScrollQuestions.tsx:17]: As I see this component is similar to SelectableQuestion, can you reuse it by passing different styles

File: src/types/quizzes/interfaces/quizzes.interface.ts [hunk 7]
```diff
@@ -11,3 +17,12 @@ export interface Quiz extends CommonEntityFields {
   author: Pick<UserResponse, '_id'>
   category: Category | null
 }
+
+export interface QuizViewProps {
+  questions: Question[]
+}
+
+export interface QuizQuestionProps {
```
Comment (Tolik170) [inline src/types/quizzes/interfaces/quizzes.interface.ts:25]: we always define props interface in the component file

File: src/types/quizzes/interfaces/quizzes.interface.ts [hunk 8]
```diff
@@ -11,3 +17,12 @@ export interface Quiz extends CommonEntityFields {
   author: Pick<UserResponse, '_id'>
   category: Category | null
 }
+
+export interface QuizViewProps {
```
Comment (Tolik170) [inline src/types/quizzes/interfaces/quizzes.interface.ts:21]: same as below

File: src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles.ts (hunk 4 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles.ts]: > m: '8px 0 32px 0'

changed

File: src/containers/my-quizzes/scroll-question/ScrollQuestions.styles.ts (hunk 5 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question/ScrollQuestions.styles.ts]: > mb

rewrited

File: src/containers/my-quizzes/scroll-question/ScrollQuestions.tsx (hunk 6 above)
Comment (dmtrth25) [reply src/containers/my-quizzes/scroll-question/ScrollQuestions.tsx]: > As I see this component is similar to SelectableQuestion, can you reuse it by passing different styles?

It's make sense. 
I used this component for rendering and added conditional for render main wrapper container

File: src/types/quizzes/interfaces/quizzes.interface.ts (hunk 7 above)
Comment (dmtrth25) [reply src/types/quizzes/interfaces/quizzes.interface.ts]: > we always define props interface in the component file

moved it back to the component

File: src/types/quizzes/interfaces/quizzes.interface.ts (hunk 8 above)
Comment (dmtrth25) [reply src/types/quizzes/interfaces/quizzes.interface.ts]: > same as below

fixed

**Pull request closed without merging**